QUEST_LV_0100.tsv

```diff
@@ -1,124 +1,124 @@
+QUEST_LV_0100_20150317_000087	$Next, we will get leather from Cronewts in Lengvas Garden.{nl}Hurry, soldier.
+QUEST_LV_0100_20150317_000088	$We will win for sure this time.{nl}The Goddesses are on our side.
+QUEST_LV_0100_20150317_000089	$You are pretty good for a rookie.{nl}It's disorganized here at the moment, so you can keep those ingredients for now.{nl}
+QUEST_LV_0100_20150317_000090	$Get to know our soldiers while I make preparations for the battle.{nl}I will call you before we confront Gaigalas.
+QUEST_LV_0100_20150317_000091	$When we burn the Duckey Fat, Gaigalas will appear after smelling the odor.{nl}Lengvas Garden will be a good place to burn it.
+QUEST_LV_0100_20150317_000092	$My heart is pounding.{nl}I feel like my soul is throbbing too.
+QUEST_LV_0100_20150317_000093	$Statue of Old Owl
+QUEST_LV_0100_20150317_000094	$There's no one in this forest. Why are you here?{nl}
+QUEST_LV_0100_20150317_000095	$Gaigalas?{nl}Defeating it without any assistance will be difficult.
+QUEST_LV_0100_20150317_000096	$I have seen many deaths here.{nl}And I don't want to see any more sacrifices.
+QUEST_LV_0100_20150317_000097	$I need Kepa Raider Sap and Cronewt leather.
+QUEST_LV_0100_20150317_000098	$Why do you have that?{nl}Hand it to me and I will make a Fragment of Memory.
+QUEST_LV_0100_20150317_000099	$With this Fragment of Memory, I will revive the memories of our sleeping comrades.
+QUEST_LV_0100_20150317_000100	$It will be difficult to defeat it without the Fragment of Memory.{nl}Please bring Infroholder horns.
+QUEST_LV_0100_20150317_000101	$How come you have all these ingredients?
+QUEST_LV_0100_20150317_000102	{memo X}$Those are people who are already gone from this world. {nl}They are repeating what happened back then because of their anger for Gaigalas.
+QUEST_LV_0100_20150317_000103	{memo X}$We can calm them if we defeat Gaigalas.{nl}Please go see the Worrying Owl.
+QUEST_LV_0100_20150317_000104	$The Statue of Worrying Owl
+QUEST_LV_0100_20150317_000105	$Please help me wake my sleeping comrades and complete the magic of that day.{nl}With the Fragment of Memory.
+QUEST_LV_0100_20150317_000106	$They may become satiated.{nl}I will stand here while remembering my comrades.
+QUEST_LV_0100_20150317_000107	$You will find Gaigalas when you go to Ishimtini Access Way.{nl}Please avenge the fallen soldiers.
+QUEST_LV_0100_20150317_000108	$I was in an isolated place so I did not fall asleep.{nl}But, watching it all was painful for me.
+QUEST_LV_0100_20150317_000109	$My comrades were attacked while they were trying to help the soldiers.
+QUEST_LV_0100_20150317_000110	$Some symbols started to show on the body of Jonas after he drank the medicine.
+QUEST_LV_0100_20150317_000111	$Archaeologist Dezic
+QUEST_LV_0100_20150317_000112	$I am looking for my assistant named Irene.{nl}Can you defeat the monsters nearby so I can go look for her?
+QUEST_LV_0100_20150317_000113	$I told her not to get excited even when she finds something.{nl}She did it again.
```

Comment:
Doesn't seem to be too much change I guess, I'll put it back to 'if'. perhaps the person didn't expect her to find something? I used "when" because I thought the person expected her to find something but warned her not to go ballistics when she does.